translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,296 @@
+
+import xmltodict
+import json
+from datetime import datetime, timezone, timedelta
+import uuid
+import random
+import string
+import sys, getopt
+from jsonschema import validate
+from jsonschema import ValidationError
+import logging
+from collections import OrderedDict
+import re
+import dateutil.parser
+import translator_shared_library
```

Comment:
Because of the way the local files are imported, it makes navigation around the code base difficult. Can we import them with the full namespace so we can include navigation?  IE
```
from translator.source_code import translator_shared_library
```

This should apply to unit tests and other modules as well.

Reply:
should be fixed now